### DISCUSSION TITLE
Is there a WiFi/BLE API or plugin for Tauri 2.0?

### DISCUSSION THREAD

**Discussion by Lixuhuilll** [tauri-apps/tauri#10529] [Q&A]
My App needs to connect to a series of BLE devices and send them profiles or subscription notifications.

**Comment by FabianLars** [comment]
Not directly, no. On discord i heard of a few people using https://github.com/deviceplug/btleplug though.

**Comment by MnlPhlp** [comment]
I created a plugin based on btleplug: [tauri-plugin-blec](https://github.com/MnlPhlp/tauri-plugin-blec).
It basically wraps btleplug for all platfroms except android.
On Android it uses the tauri plugin functionality to get rid of the complicated jni setup used in btleplug.
This way adding the plugin should 'just work' on all platforms.
I tested it on windows,linux,android and mac. With IOS I am still having some problems with signing but that is probably not related to the plugin
